### ISSUE TITLE
Vacor quest description typo

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#97]
```
What steps will reproduce the problem?
1. Kill the four bandits and return to Vacor.
2. Vacor says look for Unzel to the southeast.
3. Quest log says look for Unzel to the southwest.

What is the expected output? What do you see instead?
I assume southwest is correct. Also, maybe I've just misread Vacor's monologue, 
but quite confident it was not the case.

What version of the product are you using? On what operating system?
0.6.7b2, Android 2.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `surrano` on 20 Dec 2010 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
Vacor has a few typos when giving the bandit quest. "allright" and "ofcourse".
```

Original comment by `kims...@gmail.com` on 20 Dec 2010 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
I am unable to even find the 4th bandit. I have gone thru all the screens south 
of Fallhaven and there has been only 3 bandits. I have fought and killed the 
crazy guy on the road demanding $100 several times. Where is the 4th bandit or 
is that still to come in a later map?
```

Original comment by `guidingu...@gmail.com` on 30 Dec 2010 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
Look where the busy farmers are.
```

Original comment by `SamuelPl...@gmail.com` on 30 Dec 2010 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. Southwest is indeed correct. I have updated Vacor's 
dialog now.

Additional conversation corrections are also mentioned in issue 99.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 5:50
- Changed state: **Ready**
- Added labels: **Milestone-0.6.8**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:32
- Changed state: **Fixed**
